gnomad/utils/filtering.py

```diff
@@ -529,3 +529,44 @@ def split_vds_by_strata(
     )
 
     return [hl.vds.filter_samples(vds, list(s)) for strata, s in s_by_strata.items()]
+
+
+def remove_items_from_freq(
```

Comment:
I don't know about this, for my purpose, I want to remove these two populations, how to define the items you want to keep?

Reply:
yeah, the function I am proposing makes this more general. It can be used for your purpose, and also for others. For instance, I have a use case where I want to filter the frequencies to a set of populations.